omnia.yml

```diff
@@ -50,6 +50,9 @@
 - name: Import playbook to install ROCm on nodes
   ansible.builtin.import_playbook: utils/rocm_installation.yml
 
+- name: Verify Gaudi nodes
+  ansible.builtin.import_playbook: utils/verify_intel_gaudi/verify_intel_gaudi_installation.yml
+
```

Comment:
how much time it will need for each node?

Reply:
By default, this will take 2-3mins but if the user choses to do extensive test (change the variable) it will take around 10mins